web/src/js/components/FlowTable/FlowColumns.jsx

```diff
@@ -164,29 +165,62 @@ export function TimeStampColumn({flow}) {
 TimeStampColumn.headerClass = 'col-timestamp'
 TimeStampColumn.headerName = 'TimeStamp'
 
-export function QuickActionsColumn({flow, selected}) {
+export function QuickActionsColumn({flow, intercept, updateSettings, removeFlow, exportFlow}) {
+    const InterceptThis = useCallback((example) => {
```

Comment:
Let's use the `useDispatch` hook here: https://react-redux.js.org/tutorials/quick-start#use-redux-state-and-actions-in-react-components. This allows us to keep keep `FlowRow` "clean" and we don't need to specify all actions in FlowRow.jsx. 😃 

For `state.settings.intercept` it's a bit more tricky as we want FlowRow to be cached/memoized. If we use `useSelector` in a child component, my understanding is that the child component will not be rerendered on changes (because the parent FlowRow is cached), so we cannot do that. I think we can however move the logic from `InterceptThis` into an action creator like this:

`ducks/settings.js`
```javascript
export function addInterceptFilter(e) {
    return (dispatch, getState) => {
        let intercept = getState().settings.intercept;
        // logic here
        dispatch(updateSettings({ intercept }));
    }
}    
```

Then we can just `dispatch(addInterceptFilter(...))` here and don't need any state. Does that make sense to you? Otherwise ping me on Slack and I'll try to explain it better. :)